Move business support to own controller

davidbasalla opened the pull request:
This PR moves business support pages to their own controller, following the same pattern as Help pages (#1036).

For https://trello.com/c/phpe3PAO/546-refactor-frontend-5

emmabeynon commented on test/integration/business_support_test.rb:
```diff
@@ -0,0 +1,81 @@
+      assert_current_url "/business-support-example?edition=5"
+    end
+
+    should "render an answer in Welsh correctly" do
```
Should this be 'business support' instead of 'answer'?

emmabeynon commented on app/controllers/business_support_controller.rb:
```diff
@@ -0,0 +1,34 @@
+
+class BusinessSupportController < ApplicationController
+  before_filter -> { set_expiry unless viewing_draft_content? }
+  before_filter :redirect_if_api_request
```
I think in previous pull requests we decided to redirect before setting the expiry.

emmabeynon commented on test/functional/business_support_controller_test.rb:
```diff
@@ -0,0 +1,40 @@
+  context "GET show" do
+    setup do
+      @artefact = artefact_for_slug('business-support-example')
+      @artefact["format"] = "answer"
```
Should this be 'business-support' instead of 'answer'?

davidbasalla commented:
Thanks for the review @emmabeynon, I've fixed and rebased. Added another commit to reorder the before_filters for all of our new controllers so that the api redirect always happens before we set the expiry headers.
